### ISSUE TITLE
Add easy-to-use contact form button + text for users to report issues

### ISSUE THREAD

**Issue by joatu** [joatuapp/joatu-django#90]
We need to discuss where/how this should exist. But it would be great if somebody can very easily type a very quick message that is emailed directly to me.

**Comment by KritikaSharmaKS** [comment]
Hey, I assigned this task to myself, but as this is my first task, can you please elaborate on this?

**Comment by joatu** [comment]
Right. Well, since this is the beta that we're giving people, maybe we need a help button in the list on the left, and that would lead to a page where they can fill out a simple contact form that sends to my email (but will eventually link internally to an administrator who can respond in messages).

What do you think?

**Comment by KritikaSharmaKS** [comment]
yeah okay, sure we should do that. I'll get started.
